_posts/2023-06-20-trino-fest-2023-recap.md

```diff
@@ -6,14 +6,13 @@ excerpt_separator: <!--more-->
 image: /assets/blog/trino-fest-2023/trino-fest.png
 ---
 
-Last week we held Trino Fest, and we did not find time to chill at the
-lakehouse. Despite the theme of a summercamp for lakehouse analytics, the fest
-kept us all busy. Great demos, amazing announcements, new plugins, and use cases
-reached our active audience. Thanks go to our event host and organizer
-[Starburst](https://www.starburst.io/), our sponsors
-[AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), and many
-well-prepared speakers, and our great live audience. Now you get a chance to
-catch up on anything you missed.
+Last week we held Trino Fest, and it kept us all so busy, we forgot to spend
```

Comment:
Unrelated changes for "Remove invalid AWS logos"

Reply:
Well ... the logo is used in the post and they are separate commits

Reply:
This commit ("Remove invalid AWS logos") has changes that have nothing to do with removing the logo. Why did this paragraph change? If you look below, there are some other paragraphs with grammar changes that are unrelated to removing the logo.